docs/prerequisites.md

```diff
@@ -1,5 +1,13 @@
 # Prerequisites
 
+## Node name conventions
+
+AWS supports two naming conventions: [IP-based or resource-based naming](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-naming.html).
+
+When _IP-based naming_ is used, the nodes must be named after the instance followed by the regional domain name (`ip-xxx-xxx-xxx-xxx.ec2.<region>.internal`). If you have custom domain name set in the DHCP options, you must set `--hostname-override` on kube-proxy and kubelet to match the above-mentioned naming convention.
```

Comment:
> `ip-xxx-xxx-xxx-xxx.ec2.<region>.internal`

Is this correct? Should this be `ip-xxx-xxx-xxx-xxx.<region>.compute.internal` (for non us-east-1 and `ip-xxx-xxx-xxx-xxxx.ec2.internal` for us-east-1?

Reply:
Yeah you are correct. This one mixed up the us-east-1 convention with the other region.